spec/controllers/uploads_controller_spec.rb

```diff
@@ -24,6 +24,12 @@
     context 'with valid data' do
```

Comment:
Should we add a context for invalid data to ensure no one can upload and create their own subject? 

Also from the user's perspective, how do they know what subjects they can choose from. Maybe we should update a help text or template? (if you did so already disregard, I'm still reviewing).

Reply:
There are several ways to know. On a UI input, the list is obviously there. When creating the CSV, they can reference the subject filter list. And if they include a bad item, the error message lists all the valid ones.